app/config.yml.erb

```diff
@@ -1,33 +0,0 @@
-:base_path: <%= "#{ENV['GEMSTASH_HOME']}/data" %>
```

Comment:
We should put this back.

Reply:
🤒 ... this is true!

Reply:
https://github.com/Intellection/docker-gemstash/pull/7#issuecomment-697246840
☝️ As suggested above, I was going to put it in the config map.

Reply:
Oops, I'm referring to the file itself.

Reply:
Right right! 🤕

Reply:
I blame the pain meds! 😅